tests/playwright/sanity/includes/widgets/tabs.test.ts

```diff
@@ -30,15 +30,24 @@ test.describe( 'Tabs widget tests', () => {
 	test( 'Tabs widget sanity test', async ( { page, apiRequests }, testInfo ) => {
 		const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
 		const editor = new EditorPage( page, testInfo );
-		const contentTab = new Content( page, testInfo );
 		const tabText = 'Super tab content test';
 		const newTabTitle = 'Super test tab';
 		const defaultText = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis, pulvinar dapibus leo.';
 
 		await wpAdmin.openNewPage();
 		await editor.closeNavigatorIfOpen();
 		await editor.addWidget( 'tabs' );
-		await contentTab.addNewTab( newTabTitle, tabText );
+
+		const itemCount = await page.locator( EditorSelectors.item ).count();
+		await page.getByRole( 'button', { name: 'Add Item' } ).click();
+		await page.getByRole( 'textbox', { name: 'Title' } ).click();
+		await page.getByRole( 'textbox', { name: 'Title' } ).fill( newTabTitle );
+
+		const textEditor = page.frameLocator( EditorSelectors.tabs.textEditorIframe ).nth( itemCount );
+		await textEditor.locator( 'html' ).click();
+		await textEditor.getByText( 'Tab Content' ).click();
+		await textEditor.locator( EditorSelectors.tabs.body ).fill( tabText );
+
```

Comment:
Why?

Reply:
It was done to eliminate the `Content` class, in the future.

The `addNewTab()` method is not a general method which can be used in other places. This is a very unique use case designed for the Tabs Widget, which has a specific repeater with a Text Editor control.

Reply:
Extracting this to a separate function (could be in the test file) makes things more readable though

Reply:
Done.